### PR TITLE
Add test-cmd test to validate that we can create using a stored template

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees ptal

**Comment by csrwng** [comment]
[test]

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -386,6 +386,8 @@ osc get template ruby-helloworld-sample
 [ "$(osc new-app ruby-helloworld-sample -o yaml | grep MYSQL_PASSWORD)" ]
 [ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_USERNAME)" ]
 [ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_PASSWORD)" ]
+# check that we can create from the template without errors
+osc new-app ruby-helloworld-sample
```
Comment (bparees) [inline hack/test-cmd.sh]: should do a delete after this to clean it up.

Comment (kargakis) [inline hack/test-cmd.sh]: > should do a delete after this to clean it up.

+1

Comment (csrwng) [inline hack/test-cmd.sh]: delete added

**Comment by bparees** [comment]
Lgtm. [merge]

**Pull request merged**